core/src/main/scripts/importer/validateData.py

```diff
@@ -1375,6 +1390,36 @@ def checkEndPosition(self, value):
         # if no reasons to return with a message were found, return valid
         return True
 
+    def checkDriver(self, value):
+        """Validate the values in the cbp_driver column."""
+        if value not in ('PUTATIVE_PASSENGER', 'PUTATIVE_DRIVER', 'NA', 'Unknown'):
```

Comment:
Please make it `Putative_Passenger` and `Putative_Driver` so that it at least looks like MAF format. As I said in my previous comment: case sensitivity is annoying, so it would also be nice if the values are validated and imported case insensitive, but saved in a fixed format. We can always implement this vali/imp enhancement at a later time, but the first part not, because that will be in the data.